### ISSUE TITLE
[CAIA Intake] Pension Benefits Form: Direct Deposit Design and Content Updates

### ISSUE THREAD

**Issue by gmaciolek** [department-of-veterans-affairs/va.gov-team#77464]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: `Pension`
- **OCTO product owner:** `Emily Theis`
- **Product name:** `Veterans Pension benefits form`
- **Product manager:**  `Vicky Bellissimo` `Buddhima Kurukulasuriya`
- **Slack channel:** [benefits-pension](https://dsva.slack.com/archives/C05MS96P6P2)
- **Dedicated content writer on your team (if you have one):** We don't have a dedicated content writer on our team, but we've been working with @katherine-fung so far as our CAIA content person
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

We're adding content to the direct deposit information step of the Veterans pension form, related to recent direct deposit changes.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [x] Yes
- [ ] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

We're drafting content and creating updated designs in anticipation of the April 20th deadline described on the[ Change your VA direct deposit information ](https://va.gov/change-direct-deposit/)

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [x] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `xxxxxx`

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Figma design](https://www.figma.com/file/9JKK5Eo43uJWEr66JPiebc/Pension-Claim-Form-21P-527EZ?type=design&node-id=2146%3A2512&mode=design&t=Nre72rKdLAMo81Xt-1) 

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [x] Initial Review - @coforma-terry and @strelichl
- [x] Schedule kick-off meeting with team 3/6/2024
```

```[tasklist]
### CAIA a11y tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/77913
- [x] Attend Kickoff 3/6/2024
- [x] https://github.com/department-of-veterans-affairs/va.gov-team/issues/77898
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/77900
```

```[tasklist]
### CAIA content tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/77800
```

```[tasklist]
### CAIA IA tasks
```

**Comment by gmaciolek** [comment]
Hi @coforma-terry, Sorry I missed this the other day! I'm booked from 12-1:30 on Wednesday, but any time earlier or later works. (And, if 12PM ET is what works best on your end, I can make that work.)

**Comment by katherine-fung** [comment]
Hey @gmaciolek @vbellissimo @buddhima-coforma, 

I see that our kickoff call is tomorrow at 2pm ET. Looking forward to it! 

For context, here's a write up about the CAIA kickoff process. It also includes some info about "What is CAIA?" and who the team members are: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/Ops/kickoff-process.md

**Comment by gmaciolek** [comment]
Update re: timing/priority

From Julie Strothman:

> We’re hoping to get content added to the 527 and 526 by 3/20-ish. While the change in payments won’t occur until 4/20, this change affects everyone who submits a claim before then as well.
> 
> For example:
> 
> - I currently receive GI Bill payments in the checking acct I use for auto bill pay
> - I submit a 526 next week and list my savings account for direct deposit
> - I get an award and without warning my GI Bill payments are switched to my savings account
> - I incur late or overdraft fees because I wasn’t aware that my GI Bill payments stopped going to my auto-pay checking account

**Comment by gmaciolek** [comment]
Dropping in with an additional request from Julie Strothman: She has asked that we also add copy to the confirmation page of the form. I think it could reiterate whatever we end up saying on the direct deposit page about the direct deposit/bank account change.

The confirmation page is the last thing the user sees after submitting the form and they’re also prompted on that page to print it for their records, so it does seem like a good idea to reiterate the impact that updating their bank account information in the pension form may have on their other benefit payments.

I added [the confirmation page to the designs in Figma](https://www.figma.com/file/9JKK5Eo43uJWEr66JPiebc/Pension-Claim-Form-21P-527EZ?type=design&node-id=2169%3A2603&mode=design&t=5977Boa78ufW48Bd-1) with a shot-in-the dark idea of how/where to include this information. Please let me know if this late-breaking request can't be included in this effort by 3/11. Thank you!

**Comment by katherine-fung** [comment]
Thanks @gmaciolek! 

A question about the direct deposit content on the confirmation page: 

Does the team plan to show the content to all pension applicants, or just people who have entered bank deposit information? If we show it to everyone, we'll think about addressing just the people who entered accounts.

**Comment by shannonkford** [comment]
> Thanks @gmaciolek!
> 
> A question about the direct deposit content on the confirmation page:
> 
> Does the team plan to show the content to all pension applicants, or just people who have entered bank deposit information? If we show it to everyone, we'll think about addressing just the people who entered accounts.

@katherine-fung and @gmaciolek I don't know if you chatted about this already, but we are hoping the confirmation page might catch the few people who would have started the form before the change, set it aside, and then returned to the form after the change, but who also do not revisit the DD page again.

**Comment by gmaciolek** [comment]
@coforma-terry The updated designs and copy have been approved and our engineers are going to start working on implementing the changes. 

Do you have guidance on what the URLs should be for the 2 new pages that have been added with these updates?

**Comment by gmaciolek** [comment]
Thanks for the guidance! With that in mind, I put together the following. Let me know what you think of these URLs:

- 6.1: /pension/application/527EZ/additional-information/direct-deposit
- 6.2a: /pension/application/527EZ/additional-information/account-information
- 6.2b: /pension/application/527EZ/additional-information/other-payment-options

The numbers reference the [frame titles for each page in the Figma file](https://www.figma.com/file/9JKK5Eo43uJWEr66JPiebc/Pension-Claim-Form-21P-527EZ?type=design&node-id=2452%3A9583&mode=design&t=aM4zKB9d50Yft6JB-1).

**Comment by jennymayoco** [comment]
@gmaciolek thank you for sharing the URLs! I'm taking a look at these with our other IA specialist and can share back feedback tomorrow. Would that work?

**Comment by gmaciolek** [comment]
That's perfect, thanks!

**Comment by jennymayoco** [comment]
@gmaciolek can you tell me more about the decision to add `/additional-information` to the URL? Is it for tracking analytics?

The reason for my question is because adding `/additional-information `without a page that has info is creating an empty-subdirectory. So if a user were to go to `/pension/application/527EZ/additional-information`, they would get a 404. Based on this, we'd recommend removing that segment from the URL and so it would look like this:

6.1: /pension/application/527EZ/direct-deposit
6.2a: /pension/application/527EZ/account-information
6.2b: /pension/application/527EZ/other-payment-options

**Comment by gmaciolek** [comment]
@jennymayoco I don't know much about the original decision because I wasn't involved in those conversations, but I can find out more if you need that information. "Additional information" is "chapter" 6 of 7 for the form. Chapter 6 includes Direct deposit, Uploading supporting documents, and a question about fully developed claims. (I'm not sure if Chapter is the right form terminology for the different sections of a form.)

**Comment by jennymayoco** [comment]
@gmaciolek thanks for the background! That makes sense with how "Additional information" is represented in the designs. We would recommend removing `/additional-information` from the URs as it sounds like "additional information" is the name of the section title for a group of questions. Since there isn't any info on this page `/pension/application/527EZ/additional-information/`, it would create an empty subdirectory and users would get a 404

**Comment by gmaciolek** [comment]
This would be a departure from how all of our other URLs are structured. I will check with my team to see if what you're suggesting aligns with guidance in our collaboration cycle reviews that we haven't implemented yet. 

Can we implement the URLs with `/additional-information` included for the purposes of making this update and then revisit whether or not we need to update the URL structure of every form page to remove the section title for groups of questions

**Comment by jennymayoco** [comment]
@gmaciolek would we be able to get a list of the pension form URLs?

**Comment by gmaciolek** [comment]
@jennymayoco Here you go:

/introduction
applicant/information
applicant/mail-address
applicant/contact
military/history
military/general
military/general/add
military/pow
medical/history/age
medical/history/social-security-disability
medical/history/condition
medical/history/nursing-home
medical/history/nursing/medicaid
medical/history/nursing/medicaid/status
medical/history/monthly-pension
medical/history/va-treatment
medical/history/va-treatment/medical-centers
medical/history/federal-treatment
medical/history/federal-treatment/medical-centers
employment/current
employment/current/history
employment/previous/history
household/marital-status
household/marriage-info
household/marriages/:index
household/spouse-info
household/marital-status/separated
household/marital-status/separated/spouse-address
household/marital-status/separated/spouse-monthly-support
household/marital-status/spouse-marital-history
household/marital-status/spouse-marriages
household/dependents
household/dependents/add
household/dependents/children/information/:index
household/dependents/children/inhousehold/:index
household/dependents/children/address/:index
financial/total-net-worth
financial/net-worth-estimation
financial/transferred-assets
financial/home-ownership
financial/home-ownership/acres
financial/home-ownership/acres/value
financial/land-marketable
financial/receives-income
financial/income-sources
financial/care-expenses
financial/care-expenses/add
financial/medical-expenses
financial/medical-expenses/add
additional-information/direct-deposit
additional-information/supporting-documents
additional-information/document-upload
additional-information/faster-claim-processing
/review-and-submit
/confirmation

**Comment by EvanAtCoforma** [comment]
We developed a [design review](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77900#issuecomment-2258597676) for the Direct Deposit figma screens located in [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77900). Please let me know if you have any questions or issues with the review so I can provide clarifications or updates. Thanks!

cc: @NaomiPMC | @strelichl | @emilytheis | @gmaciolek | @sara-amanda

**Comment by EvanAtCoforma** [comment]
Also, if anyone has any awareness as to whether these screens will be developed into a prototype for accessibility auditing and testing, please let me know. Thanks!
